standard/interfaces.md

```diff
@@ -230,92 +230,92 @@ interface_member_declaration
-        Console.WriteLine($"IA.P = {((IA)c).P}");  // cast needed
-        Console.WriteLine($"IB.P = {((IB)c).P}");  // cast needed
-    }
-}
-```
-
-Within the interfaces `IA` and `IB`, member `M` is accessible directly by name. However, within method `Start`, we cannot write `c.M()` or `c.P`, as those names are not visible. To find them, casts to the appropriate interface type are needed.
+> *Example*: Consider an interface `IA` with an implementation for a member `M`. As `M` is not abstract, outside that interface or any interface derived from it, that name is not visible. It must be accessed through a reference whose compile-time type is an interface that is implicitly convertible to `IA`.
+>
+> <!-- Example: {template:"standalone-console", name:"InterfaceMember", expectedOutput:["IB.M", "IA.P = 10", "IB.P = 20"]} -->
+> ```csharp
+> interface IA
+> {
+>     public int P { get { return 10; } }
+>     public void M()
+>     {
+>         Console.WriteLine("IA.M");
+>     }
+> }
+> 
+> interface IB: IA
+> {
+>     public new int P { get { return 20; } }
+>     void IA.M()
+>     {
+>         Console.WriteLine("IB.M");
+>     }
+> }
+> 
+> class C: IB { }
+> 
+> class Test
+> {
+>     public static void Main()
+>     {
+>         C c = new C();
+>         ((IA)c).M();                               // cast needed
+>         Console.WriteLine($"IA.P = {((IA)c).P}");  // cast needed
+>         Console.WriteLine($"IB.P = {((IB)c).P}");  // cast needed
+>     }
+> }
+> ```
+>
+>Within the interfaces `IA` and `IB`, member `M` is accessible directly by name. However, within method `Main`, we cannot write `c.M()` or `c.P`, as those names are not visible. To find them, casts to the appropriate interface type are needed. *end example*
+<!-- markdownlint-disable MD028 -->
 
+<!-- markdownlint-enable MD028 -->
 > *Note*: See how the declaration of `M` in `IB` uses explicit interface implementation syntax. This is necessary to make that method override the one in `IA`; the modifier `override` may not be applied to a function member. *end note*
 
 ### §interface-fields Interface fields
 
-This clause augments the description of fields in classes [§15.5](classes.md#155-fields) for fields declared in interfaces. *field_declaration* is described in [§15.5.1](classes.md#1551-general).
+This clause augments the description of fields in classes [§15.5](classes.md#155-fields) for fields declared in interfaces.
 
-It is a compile-time error for *field_declaration* to declare an instance field.
+Interface fields are declared using *field_declaration*s ([§15.5.1](classes.md#1551-general)) with the following additional rules:
 
-As a static *field_declaration* is considered to have a default implementation ([§18.1](interfaces.md#181-general)), it is *not* abstract.
+- It is a compile-time error for *field_declaration* to declare an instance field.
+- As a static *field_declaration* has an implementation ([§18.1](interfaces.md#181-general)), it is *not* abstract.
```

Comment:
Does the language have a concept of abstract fields at all? I didn't think so. I don't believe the metadata format supports declaring an abstract field either. If this statement is not in contrast to anything, can it go without saying?